main.go

```diff
@@ -15,7 +15,7 @@ func main() {
 		Usage: "Trace OS events and syscalls using eBPF",
 		Action: func(c *cli.Context) error {
 			if c.Bool("list") {
-				printList()
+				fmt.Println(tracee.EventsIDToName)
```

Comment:
I think we should keep the separation between syscalls and other events when printing

Reply:
fixed in 7b9a03445e99c03999c3fa4bc922063eeba87111